### PR TITLE
add wei's handle pin to map

### PR DESCRIPTION
**Pull request by NetWeiChen**
add wei 's first commit! hooray!

### CODE REVIEW

**Comment by NetWeiChen** [comment]
do anothe rfix

**Comment by NetWeiChen** [comment]
fix json file

**Comment by NetWeiChen** [comment]
fix json file

**Comment by NetWeiChen** [comment]
see how it works by copying othe rjson file

File: NetWeiChenBranch.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle: netweichenbranch
+latitude: 37.774929
+longitude:`-122.419416
```
Comment (hectorsector) [inline NetWeiChenBranch.json]: @NetWeiChen great start! this back tick ` shouldn't be here, though!

**Comment by hollenberry** [comment]
Hey @NetWeiChen  ! You're so close to finishing this PR! As noted by Hector above, if you make changes to the backtick, you'll be set. 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@NetWeiChen I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/4827/commits/811d75c1d5bfc8d1391a962ad275bbbbb397703f. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**